metadata/RG351V/themes.cfg

```diff
@@ -1,25 +1,8 @@
-Art-Book https://github.com/anthonycaccese/es-theme-art-book-3-2
-Airen https://github.com/anthonycaccese/es-theme-airen
-Horizontal-Szalik https://github.com/szalik-rg351/Horizontal-Szalik
-Epic-Szalik https://github.com/szalik-rg351/Epic-Szalik
-351ELEC-351Noir https://github.com/szalik-rg351/es-theme-351elec-351noir
-351ELEC-Alekfull https://github.com/szalik-rg351/es-theme-351elec-alekfull
-351ELEC-Comicbook https://github.com/szalik-rg351/es-theme-351elec-comicbook
-351ELEC-CRAZYBOY https://github.com/szalik-rg351/es-theme-351elec-crazyboy
-351ELEC-Carbon https://github.com/szalik-rg351/es-theme-351elec-carbon
-351ELEC-Crystal https://github.com/szalik-rg351/es-theme-351elec-crystal
-351ELEC-Switch https://github.com/szalik-rg351/es-theme-351elec-switch
-351ELEC-Pixel https://github.com/szalik-rg351/es-theme-351elec-pixel
-351ELEC-Chicuelo https://github.com/szalik-rg351/es-theme-351elec-chicueloap
-351ELEC-GBZ https://github.com/szalik-rg351/es-theme-351elec-gbz
-351ELEC-Fundamental https://github.com/szalik-rg351/es-theme-351elec-fundamental
-351ELEC-CosmicRise https://github.com/szalik-rg351/es-theme-351elec-cosmicrise
-351ELEC-SuperSweet https://github.com/szalik-rg351/es-theme-351elec-supersweet
-351ELEC-Minimal https://github.com/szalik-rg351/es-theme-351elec-minimal
-351ELEC-EpicCody https://github.com/szalik-rg351/es-theme-351elec-epic-cody
-351ELEC-Roleta https://github.com/szalik-rg351/es-theme-351elec-roleta
-351ELEC-Supreme https://github.com/szalik-rg351/es-theme-351elec-supreme
-351ELEC-SpinMaster https://github.com/szalik-rg351/es-theme-351elec-spin-master
-Handheld-Simple https://github.com/Rican7/es-theme-handheld-simple
-Alt-Book https://github.com/bebeidon/es-theme-alt-book-3-2
-# This line will be ignored but its needed heheh
+V-Book https://github.com/szalik-rg351/351elec-rg351v-alt-book
+Epic-Cody https://github.com/szalik-rg351/351elec-rg351v-epic-cody
+Minimal https://github.com/szalik-rg351/es-theme-351elec-minimal
+GBZ https://github.com/szalik-rg351/351elec-rg351v-gbz
+Cody-Destroy https://github.com/szalik-rg351/351elec-rg351v-cody-destroy
+Art-Book-Not-Supported-Yet https://github.com/anthonycaccese/es-theme-art-book-3-2

```

Comment:
```suggestion
Art-Book https://github.com/anthonycaccese/es-theme-art-book-3-2
```

Reply:
Please remove "Not Supported" from this and Handheld Simple.  The themes work and display ok.